app/src/main/java/com/goayo/debtify/view/AddDebtFragment.java

```diff
@@ -0,0 +1,77 @@
+package com.goayo.debtify.view;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.databinding.DataBindingUtil;
+import androidx.fragment.app.Fragment;
+import androidx.navigation.fragment.NavHostFragment;
+import androidx.recyclerview.widget.GridLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.goayo.debtify.R;
+import com.goayo.debtify.databinding.AddDebtFragmentBinding;
+import com.goayo.debtify.databinding.PickUsersFragmentBinding;
+import com.goayo.debtify.model.ModelEngine;
+import com.goayo.debtify.modelaccess.IUserData;
+import com.goayo.debtify.view.adapter.PickUserAdapter;
+import com.goayo.debtify.view.adapter.UserCardViewAdapter;
+
+import java.util.Set;
+
+/**
+ * @author Alex Phu, Yenan Wang
```

Comment:
Author should be @gabrielbrattgard name? Not my name.

Reply:
Done, changed it and added correct header comment.